### ISSUE TITLE
Remove --device=all

### ISSUE THREAD

**Issue by jbruechert** [flathub/info.mumble.Mumble#71]
I can't find the reason for adding this permission initially, and I would like to remove it again.
If you happen to know what this is used for, let me know. Otherwise I'll just remove it in a few days.

**Comment by maweil** [comment]
I just tried disabling it via Flatseal and Mumble still works fine for me. So I also think this could be removed.

Furthermore, I just noticed the `--filesystem=home` permission being enabled for Mumble (which I also deactivated now). Can we remove that as well or was there a specific reason for adding it?

**Comment by maweil** [comment]
Hi @JBBgameich, thanks for clarification where this comes from. 

I'm not sure though whether this is still the case though. I'm not a QT expert, but it seems the support for the portals is built into the framework. I just verified that I have the home permission disabled in Flatseal and did two recordings using this Flatpak app nonetheless. The file path selection worked, it will just display different path name than initially selected as can be seen on this screenshot: 
![screenshot_mumble_record_without_home](https://github.com/flathub/info.mumble.Mumble/assets/16721506/94ac1614-0a39-4e8c-8e0a-a29e09bc7c5c)

But this I also saw in other flatpak apps. Saving the recordings worked nonetheless and they appeared in the folder I initially selected.
